code/modules/virus2/effect/stage_2.dm

```diff
@@ -698,3 +751,14 @@
 		H.vomit(instant = 1)
 	else
 		H.vomit()
+
+/datum/disease2/effect/antitox
+	name = "Antioxidantisation Syndrome"
+	desc = "A very real syndrome beloved by Super-Food Fans and Essenstial Oil Enthusiasts; encourages the production of anti-toxin within the body."
```

Comment:
"Essential" eh?

Reply:
Essenstial

Reply:
This was a test. well done for spotting the mistake.  (this is definitely not a cope)

Reply:
I'd keep it because it fits with the theme of the disease, but a typo is a typo.